src/ol_openedx_rapid_response/README.rst

```diff
@@ -0,0 +1,67 @@
+ol_openedx_canvas_integration
+=============================
+
+Overview
+--------
+
+A django app plugin for edx-platform to enable ``Rapid Response`` reports download functionality in ``Instructor`` tab.
+
+
+**NOTE:**
+
+Since the edx-platform’s ``Instructor Dashboard`` does not support the plugin based tab extensions so we had to make some changes in the edx-platform itself to add ``Rapid Responses`` tab through this plugin.
+So, the ``edx-platform`` branch/tag you're using must include below commit for ``ol-openedx-rapid-response`` plugin to work properly:
+
+- https://github.com/mitodl/edx-platform/commit/f6c7cc6ea528819e8779e3137c806911a81a3198
+
+Installation
+------------
+
+You can install it into any Open edX instance by using any of the following methods:
+
+- To get the latest stable release from PyPI
+
+
+.. code-block::
+
+    pip install ol-openedx-rapid-response
```

Comment:
This can be a bit more descriptive and better organized:

```suggestion
**Option 1: Install from PyPI

.. code-block::

    # If running devstack in docker, first open a shell in lms (make lms-shell)
    
    pip install ol-openedx-rapid-response
```